library/js/xl/formatting.js

```diff
@@ -76,10 +76,11 @@
         const format = typeof date_display_format !== 'undefined' ? date_display_format : 0;
 
         let formatted;
+        // uses php-date-formatter library which in turn uses the php datetime format @see https://www.php.net/manual/en/datetime.format.php
         if (format === 1) {
-            formatted = seconds ? "h:mm:ss a" : "h:mm a";
-        } else { // (format === 0)
-            formatted = seconds ? "HH:mm:ss" : "HH:mm";
```

Comment:
I wonder if I did this? I had to fix jsGlobal out of scope I think for portal appointments using globals date format

Reply:
I didn't check the git blame on it as it looked like something I did lol

Reply:
ah ha! It was you:) kidding I don't know...

Reply:
yea, git blame says it was me on the format.

Reply:
my goodness, I truly thought it was me! doesn't matter and I'll check portal for change tomorrow unless need tonight